lib/stdlib/src/proc_lib.erl

```diff
@@ -522,6 +524,30 @@ translate_initial_call(DictOrPid) ->
 	    {?MODULE,init_p,5}
     end.
 
+%% -----------------------------------------------------
+%% [get] set_process_description/1
+%% Add and fetch process id's to aid in debugging
+%% -----------------------------------------------------
+
+-spec set_process_description(Id :: term()) -> ok.
+set_process_description(Id) ->
+    put('$process_description', Id),
```

Comment:
isn't all data stored in dictionary a process data? is `process_` prefix adding information?
for example, we don't have `process_initial_call` ...

Reply:
No but the API function is not get_process_initial_call either, I do want to be explicit here.

Reply:
I'm saying `process_description` could maybe be just `description` (or whatever we like) without loosing any information.